controllers/module_reconciler.go

```diff
@@ -298,6 +316,48 @@ func (r *ModuleReconciler) checkImageExists(ctx context.Context, mod *kmmv1beta1
 	return imageAvailable, nil
 }
 
+func (r *ModuleReconciler) handleDriverContainerServiceAccount(ctx context.Context, mod *kmmv1beta1.Module) error {
+	logger := log.FromContext(ctx)
+
+	sa := &v1.ServiceAccount{}
+	fmt.Println("Before rbacAPI.GetDriverContainerServiceAccount")
+	err := r.rbacAPI.GetDriverContainerServiceAccount(ctx, *mod, sa)
```

Comment:
Why do we need to get the ServiceAccount first?

Reply:
Very good point, too hasty of a decision there! I will change it.